jguerrara_lru_cache.py

```diff
@@ -0,0 +1,26 @@
+class LRUCache:
+
+    def __init__(self, capacity):
+        self.vals = dict()
+        self.max_size = capacity
+
+    def put(self, key, value):
+        if key in self.vals:
+            del self.vals[key]
+            self.vals[key] = value
+        else:
+            if len(self.vals) < self.max_size:
+                self.vals[key] = value
+            else:
+                del self.vals[list(self.vals)[0]]
+                self.vals[key] = value
+
+    def get(self, key):
+        if key in self.vals:
```

Comment:
Don't need to do a check for key before returning the value. Instead you can directly return the value in the dictionary